content/tutorials/manage-account-settings/offline-account-setup.md

```diff
@@ -0,0 +1,332 @@
+         "Sequence" : 1,
+         "index" : "13F1A95D7AAB7108D5CE7EEAF504B2894B8C674E6D68499076441C4837282BF8"
+      },
+      "ledger_current_index" : 4,
+      "status" : "success",
+      "validated" : false
+   }
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+### {{n.next()}}. Enter the sequence number on the offline machine.
+
+Save the account's starting sequence number on the offline machine as its current sequence number. Whenever you sign a new transaction using the offline machine, you will use the current sequence number, then increase the current sequence number by 1.
+
+You can prepare several transactions in advance this way, then transfer the signed transactions to the online machine all at once and submit them. As long as each transaction is validly formed and pays a sufficient [transaction cost](transaction-cost.html), the XRP Ledger network should eventually include those transactions in validated ledgers, keeping the account's sequence number in the shared XRP Ledger in sync with the "current" sequence number you are tracking on the offline machine.
+
+
+
+### {{n.next()}}. Sign initial setup transactions, if any.
+
+On the offline machine, prepare and sign transactions for configuring your account. The details depend on how you intend to you use your account. Some examples of things you might want to do include:
+
+- [Assign a regular key pair](assign-a-regular-key-pair.html) that you can rotate regularly.
+- [Require destination tags](require-destination-tags.html) so that users can't send you payments without tagging the reason they sent it or the customer it's intended for.
+- [Set Up Multi-Signing](set-up-multi-signing.html) for a higher bar of account security.
+- [Enable DepositAuth](depositauth.html) so you can only receive payments you've explicitly accepted or from parties you've pre-approved.
+- [Enable RequireAuth](become-an-xrp-ledger-gateway.html#enabling-requireauth) so that users can't open [trust lines](trust-lines-and-issuing.html) to you without your permission. If you don't plan to use the XRP Ledger's decentralized exchange or issued currency features, you may want to do this as a precaution.
+- Issued currency [Gateways](become-an-xrp-ledger-gateway.html) may have additional setup, such as:
+    - [Set a TransferRate](become-an-xrp-ledger-gateway.html#transferrate) for users transferring your issued currencies.
+    - [Disallow XRP payments](become-an-xrp-ledger-gateway.html#disallowxrp) if you plan to use this address for issued currencies only.
+
+At this stage, you are only signing the transactions, not submitting them. For each transaction, you must provide all fields, including fields that are normally auto-fillable such as the `Fee` ([transaction cost](transaction-cost.html)) and `Sequence` ([sequence number][]). If you prepare multiple transactions at the same time, you must use sequentially increasing `Sequence` numbers in the order you want the transactions to execute.
```

Comment:
> What happens if you were to increment by 2 (for example) for each transaction? Does the next transaction (which skips a number) fail?

If you increment by 2, that would mean the XRPL is expecting sequence X, and you submit a transaction with sequence X+1. That transaction would retry (the error code is `terPRE_SEQ`) a few times on the server it was submitted to, in case X gets processed within that time. If X does not come in, then X+1 gets dropped. *However*, that drop can not be guaranteed. A node using a different implementation could easily hold on to that transaction indefinitely such that when X does get processed, it reprocesses X+1 immediately, allowing it to succeed. This is an example of the "open-ended possibilities" I described above.

> If incrementing by one, and a transaction fails, can that number ever be reused?